pkg/scheduler/backend/cache/cache.go

```diff
@@ -44,8 +44,8 @@ var (
 // "ctx" is the context that would close the background goroutine.
 func New(ctx context.Context, ttl time.Duration, apiDispatcher fwk.APIDispatcher) Cache {
 	logger := klog.FromContext(ctx)
-	cache := newCache(ctx, ttl, cleanAssumedPeriod, apiDispatcher)
-	cache.run(logger)
+	cache := newCache(logger, ttl, cleanAssumedPeriod, apiDispatcher)
+	go cache.run(ctx, logger)
```

Comment:
Don't create a goroutine here:
```suggestion
	cache.run(ctx, logger)
```